A59-lrs-named-metrics.md

```diff
@@ -0,0 +1,97 @@
+A59: xDS LRS Named Metrics Support
+----
+* Author: yousukseung
+* Approver(s): markdroth
+* Status: Ready for Implementation
+* Implemented in: C-core
+* Last updated: 2023-05-10
+* Discussion at: TBA
+
+## Abstract
+
+This proposal describes how the gRPC xDS client will support named metrics from the backends in load reports to the LRS server at the locality level.
+
+## Background
+
+The gRPC xDS client currently includes request counts in LRS load reports. These are locally counted simple request counts, however there are cases where the service needs to include custom data from the backends in LRS load reports for load balancing purposes. gRPC now supports ORCA based load reporting from the backend and this proposal describes how to integrate it with LRS load reporting in the gRPC xDS client.
+
+### Related Proposals: 
+[A51: Custom Backend Metrics Support][A51]
+[Open Request Cost Aggregation (ORCA)][ORCA]
```

Comment:
I don't think we need to specifically link to the ORCA spec, since that's already referred to in A51.

Reply:
Done. FYI I left a link to ORCA page in the background section.